mineral-ui/mineral-ui
# Fix Dropdown keyboard nav

### Description

Fixes Dropdown Keyboard nav by making Button only re-render its DOM element when the element prop is changed.  This allows the button to retain focus across renders, which is needed for Dropdown event listeners to function properly.

### Motivation and context

closes #443

### Screenshots, videos, or demo, if appropriate

https://443-dropdown-keyboard-nav--mineral-ui.netlify.com/

### How to test

1. View Dropdown demo
2. Tab to the Button in the first Dropdown example
3. Hit the up or down arrow once to open the Dropdown
4. Hit the up or down arrow again to navigate the menu items
5. View the Button demo
6. Ensure everything behaves as expected

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### Checklist
